outline.md

```diff
@@ -3,40 +3,63 @@ title: Serverless perspectives
 subtitle: An outline 
 ...
 
-I.  What is serverless?
-    A.  Elucidating tools, patterns, and practices
-        1. Functions as a Service
-            a. Variable cost application models
-        2. Utility computing
-        3. Frameworks
-            a. Serverless
-            b. Others?
-
-II.  Why should I use serverless?
-    A.  Helping you do what you know you should be doing already
-        1. API-driven development
-            a. Platforms as a first-class citizen
-            b. Meta-data analysis and informed decision making
-        2. Value chain analysis
-            a. Purposefully evolve the ecosystem
-            b. Focus on value creation
-        3. Two pizza teams
-            a. Decoupled architecture
-        4. Accelerate the feedback cycles
-            a. Owning the OODA loop
-    B.  Serverless use cases, a primer
-        1. API platforms
-            a. External
-            b. Internal
-        2. Application development
-            a. Green field
-            b. Brown field 
-        3. Automation in the cloud
-
-III.  Examples
-    A.  Monetizing your services with AWS Marketplace and API Gateway
-    B.  Secure automated serverless delivery at scale
-    C.  Voice interactions with Alexa Skills
-    D.  Function orchestration with Step Functions
-    E.  Bakeoff: AWS Lambda Functions, Azure Functions, Google Cloud Functions, & Apache OpenWhisk
-    F.  Bakeoff pt II: Serverless Framework, Apex, & Claudia
+##  What is serverless?
+
+1. Elucidating tools, patterns, and practices
+    1. Functions as a Service
+        1. Variable cost application models
+    1. Utility computing
+    1. Frameworks
+        1. Serverless [@serverless]
+        1. SAM (Serverless Application Model)
+        1. Others?
+
+##  Why should I use serverless?
+    
+1. Helping you do what you know you should be doing already
+    1. API-driven development
+        1. Platforms as a first-class citizen
+        1. Meta-data analysis and informed decision making
+    1. Value chain analysis
+        1. Purposefully evolve the ecosystem
+        1. Focus on value creation
+1. Two pizza teams
```

Comment:
This needs to be indented - still a part of the 'what you should already by doing' list